Traceback (most recent call last):
  File "/opt/conda/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/conda/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/conda/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/conda/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/conda/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/opt/conda/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/conda/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/conda/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from ocpmodels.datasets import TrajectoryLmdbDataset, SinglePointLmdbDataset

# TrajectoryLmdbDataset is our custom Dataset method to read the lmdbs as Data objects. Note that we need to give the path to the folder containing lmdbs for S2EF
dataset = TrajectoryLmdbDataset({"src": "data/s2ef/train_100/"})

print("Size of the dataset created:", len(dataset))
print(dataset[0])
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mAssertionError[0m                            Traceback (most recent call last)
Input [0;32mIn [19][0m, in [0;36m<cell line: 4>[0;34m()[0m
[1;32m      1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mocpmodels[39;00m[38;5;21;01m.[39;00m[38;5;21;01mdatasets[39;00m [38;5;28;01mimport[39;00m TrajectoryLmdbDataset, SinglePointLmdbDataset
[1;32m      3[0m [38;5;66;03m# TrajectoryLmdbDataset is our custom Dataset method to read the lmdbs as Data objects. Note that we need to give the path to the folder containing lmdbs for S2EF[39;00m
[0;32m----> 4[0m dataset [38;5;241m=[39m [43mTrajectoryLmdbDataset[49m[43m([49m[43m{[49m[38;5;124;43m"[39;49m[38;5;124;43msrc[39;49m[38;5;124;43m"[39;49m[43m:[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mdata/s2ef/train_100/[39;49m[38;5;124;43m"[39;49m[43m}[49m[43m)[49m
[1;32m      6[0m [38;5;28mprint[39m([38;5;124m"[39m[38;5;124mSize of the dataset created:[39m[38;5;124m"[39m, [38;5;28mlen[39m(dataset))
[1;32m      7[0m [38;5;28mprint[39m(dataset[[38;5;241m0[39m])

File [0;32m/opt/ocp/ocpmodels/datasets/lmdb_dataset.py:138[0m, in [0;36mTrajectoryLmdbDataset.__init__[0;34m(self, config, transform)[0m
[1;32m    137[0m [38;5;28;01mdef[39;00m [38;5;21m__init__[39m([38;5;28mself[39m, config, transform[38;5;241m=[39m[38;5;28;01mNone[39;00m):
[0;32m--> 138[0m     [38;5;28;43msuper[39;49m[43m([49m[43mTrajectoryLmdbDataset[49m[43m,[49m[43m [49m[38;5;28;43mself[39;49m[43m)[49m[38;5;241;43m.[39;49m[38;5;21;43m__init__[39;49m[43m([49m[43mconfig[49m[43m,[49m[43m [49m[43mtransform[49m[43m)[49m
[1;32m    139[0m     warnings[38;5;241m.[39mwarn(
[1;32m    140[0m         [38;5;124m"[39m[38;5;124mTrajectoryLmdbDataset is deprecated and will be removed in the future.[39m[38;5;124m"[39m
[1;32m    141[0m         [38;5;124m"[39m[38;5;124mPlease use [39m[38;5;124m'[39m[38;5;124mLmdbDataset[39m[38;5;124m'[39m[38;5;124m instead.[39m[38;5;124m"[39m,
[1;32m    142[0m         stacklevel[38;5;241m=[39m[38;5;241m3[39m,
[1;32m    143[0m     )

File [0;32m/opt/ocp/ocpmodels/datasets/lmdb_dataset.py:50[0m, in [0;36mLmdbDataset.__init__[0;34m(self, config, transform)[0m
[1;32m     48[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39mpath[38;5;241m.[39mis_file():
[1;32m     49[0m     db_paths [38;5;241m=[39m [38;5;28msorted[39m([38;5;28mself[39m[38;5;241m.[39mpath[38;5;241m.[39mglob([38;5;124m"[39m[38;5;124m*.lmdb[39m[38;5;124m"[39m))
[0;32m---> 50[0m     [38;5;28;01massert[39;00m [38;5;28mlen[39m(db_paths) [38;5;241m>[39m [38;5;241m0[39m, [38;5;124mf[39m[38;5;124m"[39m[38;5;124mNo LMDBs found in [39m[38;5;124m'[39m[38;5;132;01m{[39;00m[38;5;28mself[39m[38;5;241m.[39mpath[38;5;132;01m}[39;00m[38;5;124m'[39m[38;5;124m"[39m
[1;32m     52[0m     [38;5;28mself[39m[38;5;241m.[39mmetadata_path [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mpath [38;5;241m/[39m [38;5;124m"[39m[38;5;124mmetadata.npz[39m[38;5;124m"[39m
[1;32m     54[0m     [38;5;28mself[39m[38;5;241m.[39m_keys, [38;5;28mself[39m[38;5;241m.[39menvs [38;5;241m=[39m [], []

[0;31mAssertionError[0m: No LMDBs found in 'data/s2ef/train_100'
AssertionError: No LMDBs found in 'data/s2ef/train_100'

